testing/src/test/java/org/motechproject/nms/testing/it/api/utils/ApiTestHelper.java

```diff
@@ -0,0 +1,17 @@
+package org.motechproject.nms.testing.it.api.utils;
+
+import org.motechproject.nms.flw.domain.FrontLineWorker;
+import org.motechproject.nms.flw.domain.FrontLineWorkerStatus;
+
+/**
+ * Api test helper with static methods
+ */
+public final class ApiTestHelper {
+
+    public static FrontLineWorker createFlw(String name, Long phoneNumber, String mctsFlwId, FrontLineWorkerStatus status) {
```

Comment:
If you're gonna do that, why not also add createFlwWithDistrictAndState and save yourself a little extra work?